libkpod/container_server.go

```diff
@@ -590,6 +597,44 @@ func (c *ContainerServer) ListContainers() []*oci.Container {
 	return c.state.containers.List()
 }
 
+// ListRunningContainers returns a list of all containers that are currently running
+func (c *ContainerServer) ListRunningContainers() []*oci.Container {
+	containers := c.ListContainers()
+	retCtrs := []*oci.Container{}
+	for _, container := range containers {
+		if container.State().Status == "running" {
+			retCtrs = append(retCtrs, container)
+		}
+	}
+	return retCtrs
+}
+
+// ListContainersFiltered returns the ids of all containers listed in the containers slice, unless the slice is empty.
+// if onlyRunning is set to true, only the ids of running containers will be returned
+func (c *ContainerServer) ListContainersFiltered(namesOrIds []string, onlyRunning bool) ([]string, error) {
+	containers := []*oci.Container{}
+	ids := []string{}
+	if len(namesOrIds) > 0 {
```

Comment:
Do you need to do something different in this if statement if onlyRunning is set?  Or is the function comment off a bit?